src/components/Home/Home.tsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react'
+import {Button, Col, Row, Carousel, Typography, Space} from 'antd';
+import findMentor from '../../assets/new-findmentor.png'
+import success from '../../assets/success.png'
+import scholarXArchive from '../../assets/scholarX-archive.png'
+import TestimonialCard from "./Testimonials/TestimonialCard";
+import MentorFAQCollapse from "./FAQ/MentorFAQCollapse";
+import MenteeFAQCollapse from "./FAQ/MenteeFAQCollapse";
+import styles from "./Home.css"
+
+const {Text} = Typography;
```

Comment:
There seems to be formatting issues, check whether the eslint is working or not. run `npm run lint`